include/cutlass/gemm/kernel/intel_pvc_gemm.hpp

```diff
@@ -258,13 +269,18 @@ class GemmUniversal<
       smem_buf,
       params.mainloop
     );
-    auto gmem_tiled_copy_c = make_xe_2d_copy<XE_2D_U32x8x16x1x1_ST_N>(make_tensor(params.epilogue.ptr_D, make_shape(M, N, L), params.epilogue.dD));
-
-    Tensor tCi = gmem_tiled_copy_c.get_pvc_tensor(make_coord(m_coord, n_coord, 0),
-                                                  make_shape(Int<FragsM>{}, Int<FragsN>{}, L),
-                                                  make_stride(Int<DpasM>{}, Int<DpasN>{}));
 
-    copy(gmem_tiled_copy_c, accumulators, tCi(_,_,_,l_coord));
+    CollectiveEpilogue epilogue{params.epilogue, shared_storage.tensors.epilogue};
+    epilogue(
+      problem_shape_MNKL,
+      subgroup_shape,
+      tile_coord,
+      accumulators,
+      tiled_mma,
+      residue_mnk,
+      thread_idx,
+      smem_buf
```

Comment:
Do you use smem_buf? if not you can pass `nullptr`

Reply:
Same here, `smem_buf` is used at the beginning of the `operator()` method.

Reply:
I think you can just remove the parameter